Formula/azure-cli.rb

```diff
@@ -1,8 +1,8 @@
 class AzureCli < Formula
   desc "Microsoft Azure CLI 2.0"
   homepage "https://docs.microsoft.com/cli/azure/overview"
-  url "https://azurecliprod.blob.core.windows.net/releases/azure-cli-2.0.20.tar.gz"
```

Comment:
decommissioned?

Reply:
The previous files will still be there. We will not remove them.
From now on though, we release based of a specific git tag so now we can just point to the github release archive. It seems this is more standard across package managers to just point to the GitHub archive if it's available.

(Previously, the .tar.gz was custom so we had to create it and host it ourselves)

Reply:
@derekbekoe you may still want to upload separate release artifacts to GitHub, even if you're just downloading the source archive tarball and reuploading it, to avoid getting caught up in this sort of situation https://github.com/Homebrew/homebrew-core/issues/18044.

Reply:
I changed the URL to https://azurecliprod.blob.core.windows.net/releases/azure-cli-2.0.21.tar.gz.
Does that work?

Reply:
Yup, that will be safer.